docs/src/Experimental/Singularities/space_germs.md

```diff
@@ -0,0 +1,208 @@
+permitting exact computations, in particular not over ``{\mathbb R}`` or 
+``{\mathbb C}``. This usually does not pose a problem, if the input data is 
+in a exact smaller field. But unfortunately, also analytic algebras themselves 
+do not allow exact computations so that applications have to be considered 
+in a localization of an affine algebra. With due care, output may again be 
+interpreted in terms of a multivariate formal power series using the following 
+inclusions:
+```math
+{\mathbb Q}[\underline{x}]_{\langle x \rangle} \hookrightarrow
+  {\mathbb Q}\{\underline{x}\}\hookrightarrow
+  {\mathbb Q}[[\underline{x}]]
+```
+At particular points, where this difficulty of interpretation 
+manifests itself prominently, a suitable warning, note or example has been 
+placed (but certainly not everywhere). 
+
+Textbooks covering space germs in the sense of analytic algebras and
+singularity theory are:
+- [GLS07](@cite)
+- [dJP00](@cite)
+
+For the point of view of schemes, we refer to the page on schemes and the references given there; an example of a standard textbook is
+- [Har77](@cite).
+
+## Creating Space Germs in OSCAR
+
+In general, space germs in Oscar are created in the following ways:
+
+ * localization of an affine scheme at a point
+
+   ```julia
+   SpaceGerm(X::AffineScheme, I::Ideal)
+   ```
+   where I is a (maximal) ideal describing the chosen kk-point on the affine kk-scheme X.  
+   **Provides:** SpaceGerm.
+   ```julia
+   germ_at-point(X::AffineScheme, I::Ideal)
+   ```
+   where I is a (maximal) ideal describing the chosen kk-point on the affine kk-scheme.  
+
+   **Provides:** SpaceGerm, restriction map.
+
+
+ * localization of a polynomial ring at a point
+
+   ```julia
+   germ_at_point(R::MPolyRing, I::MPolyIdeal)
+   germ_at_point(R::MPolyQuo, I::MPolyIdeal)
+   ```
+   where I is a maximal ideal describing the chosen base_ring(R)-point.
+
+   **Provides:** space germ and restriction map.
+
+ * localized ring with respect to complement of prime ideal or complement of maximal ideal)
+
+   ```julia
+   SpaceGerm(R::MPolyLocalizedRing)
+   SpaceGerm(R::MPolyQuoLocalizedRing)
+   ```
+   **Provides:** Space germ
+   ```julia
+   germ_at_point(R::MPolyLocalizedRing)
+   germ_at_point(R::MPolyQuoLocalizedRing)
+   ```
+
+   **Provides:** Space germ, restriction map  
+
+   (point inherited from underlying local ring)
+
+Morphisms of space germs are morphisms of the underlying local rings and hence
+also handled in this way in Oscar.
+
+## Basic functionality for space germs
+
+Most of the basic functionality immediately falls back to the underlying 
+affine algebra or affine scheme and is just provided on the geometric side 
+for convenience and consistence of functionality:
+
+### internal data of a space germ  
+
+ * Pass from the germ $(X,x)$ back to some affine scheme $X$ with the appropriate localization at $x$ by
+   ```julia
+   representative(X::SpaceGerm)
+   ```
+   **Provides:** AffineScheme   
```

Comment:
It is not completely clear what this means. In principle, a germ is already an affine scheme in the sense that it is of the form `Spec(R)` for a ring `R`. The above reads a bit as if an affine scheme was something else for you here. What do you want to return?

Reply:
I clarified this in the documentation now.